pysus/ftp/databases/pni.py

```diff
@@ -0,0 +1,94 @@
+from typing import List, Union, Optional, Literal
+
+from pysus.ftp import Database, Directory, File
+from pysus.ftp.utils import zfill_year, to_list, parse_UFs, UFs
+
+
+class PNI(Database):
+    name = "PNI"
+    paths = (
+        Directory("/dissemin/publicos/PNI/DADOS"),
+    )
+    metadata = {
+        "long_name": "Sistema de Informações do Programa Nacional de Imunizações",
+        "source": (
+            "https://datasus.saude.gov.br/acesso-a-informacao/morbidade-hospitalar-do-sus-sih-sus/",
+            "https://datasus.saude.gov.br/acesso-a-informacao/producao-hospitalar-sih-sus/",
+        ),
+        "description": (
+            "O SI-PNI é um sistema desenvolvido para possibilitar aos gestores "
+            "envolvidos no Programa Nacional de Imunização, a avaliação dinâmica "
+            "do risco quanto à ocorrência de surtos ou epidemias, a partir do "
+            "registro dos imunobiológicos aplicados e do quantitativo populacional "
+            "vacinado, agregados por faixa etária, período de tempo e área geográfica. "
+            "Possibilita também o controle do estoque de imunobiológicos necessário "
+            "aos administradores que têm a incumbência de programar sua aquisição e "
+            "distribuição. Controla as indicações de aplicação de vacinas de "
+            "imunobiológicos especiais e seus eventos adversos, dentro dos Centros "
+            "de Referências em imunobiológicos especiais."
+        ),
+    }
+    groups = {
+        "CPNI": "Centro de Parto Normal Intra-Hospitalar",
```

Comment:
It's curious that PNI has a dataset for "Parto Normal Intra-Hospitalar", does not seem related...

Reply:
By any chance, do you know the difference between CPNI and DPNI? both file prefixes are found in the ftp server